### ISSUE TITLE
no sound in video 

### ISSUE THREAD

**Issue by fr3gs** [pietje666/plugin.video.vrt.nu#62]
Firstt of all, great app! Thank you for this.
We've been using it for a while now.

**Describe the bug**
Since a few days though, we are not hearing any sound in the video's.
If i go to sound settings and check "geluidskanaal" it says none
All other non vrt.nu video's work fine.

I was on kodi 17. I just now upgraded tot 18 to check if it would resolve anything but no luck.

**To Reproduce**
1. just play a video

**(please complete the following information):**
 - Operating system: windows 10 v1809
 - Kodi Version: 17 & 18

**Additional context**
- Using a VPN: nope
- Country you are using the addon from: belgium

**Log (if available)**
[kodi.log](https://github.com/pietje666/plugin.video.vrt.nu/files/2921056/kodi.log)

any idea's?
kind regards

**Comment by pietje666** [comment]
Does this count for specific video's? or all? Does this behavior also occur for livestreams?

**Comment by fr3gs** [comment]
So far for all videos. Live stream is ok though.

**Comment by fr3gs** [comment]
Hmm sorry
Must have been a coincidence, I tried some more now and others do seem to
work. I'll check if the encoding or something is different in the videos

**Comment by pietje666** [comment]
which videos do we talk about now that does not have audio?

**Comment by pietje666** [comment]
@fr3gs i guess this can be close by now? Or do you still have issues?

**Comment by fr3gs** [comment]
Hi pietje666

Sorry for the late reply, haven't been able to test yet. It's mostly kids
show that have the problem.
Biba An Louba
George and Paul

Are 2 I know for sure that have the issue
Mega Mindy had the issue but not anymore.
I'll try to do some more testing and get back to you.
Kind regards

**Comment by mediaminister** [comment]
I investigated and fixed this issue. 
This issue was only present on a minority of streams, mainly childeren programs, when the user did not have inputstream.adaptive installed.
In that case, the streamservice class falls back on HLS selection and the HLS substream selection function. (indicated in source code as last resort stream selection).

Because VRT added separate audio uri's in their HLS-playlists, there was no audio selected in some cases. I rewrote the HLS substream selection to support separate audio streams.
